app/controllers/heritages_controller.rb

```diff
@@ -28,6 +28,13 @@ def update
     render json: @heritage
   end
 
+  def update_with_district
+    @heritage = BuildHeritage.new(permitted_params, district: @district).execute
+    @heritage.save_and_deploy!(without_before_deploy: false,
+                               description: "Update to #{@heritage.image_path}")
+    render json: @heritage
```

Comment:
can't this just be in `update` but with `district` as an optional?

Reply:
@davidsiaw ok I updated the logic to pass the value in body as optional.
Let me know what you think 🙏